core/src/test/scala/org/apache/spark/SparkContextSchedulerCreationSuite.scala

```diff
@@ -129,4 +129,29 @@ class SparkContextSchedulerCreationSuite
       case _ => fail()
     }
   }
+
+  test("bad-spark-url") {
+    val e = intercept[SparkException] {
+      createTaskScheduler("spark:*")
```

Comment:
Aren't these tests identical to existing tests above (`bad-master` and `local-default-parallelism`)?